includes/404-widget.php

```diff
@@ -0,0 +1,25 @@
+<?php
+/**
+ * LSX functions and definitions - Config.
+ *
+ * @package    lsx
+ * @subpackage 404-widget
+ */
+
+function lsx_widget_area_404_init() {
```

Comment:
@viscosho please check if the function exists before to create it. Check the other include files to see how we do it in LSX core theme.